excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportServiceImpl.java

```diff
@@ -228,9 +228,7 @@ public IdamProperties getUserDetails() {
     }
 
     private JurisdictionEntity importJurisdiction(JurisdictionEntity jurisdiction) {
-        return jurisdictionService.get(jurisdiction.getReference()).orElseGet(() -> {
-            jurisdictionService.create(jurisdiction);
-            return jurisdiction;
-        });
+        jurisdictionService.create(jurisdiction);
+        return jurisdiction;
```

Comment:
Couldn't see why we needed to do this.

Reply:
If Jurisdiction already exists in the DB without banner details, and if user tries to import new definition with banner details added into the definition file, then it is not going to save banner information into db, as the previous logic thinks jurisdiction already exists.

Reply:
done changes